sources/2023-09-19-initialize-reward-rate-configuration/initialize.move

```diff
@@ -0,0 +1,20 @@
+script {
+    use aptos_framework::aptos_governance;
+    use aptos_framework::staking_config;
+    use aptos_std::fixed_point64;
+
+    fun main(proposal_id: u64) {
+        let framework_signer = aptos_governance::resolve_multi_step_proposal(proposal_id, @0000000000000000000000000000000000000000000000000000000000000001, vector::empty<u8>());
+        let (curr_rewards_nominator, curr_rewards_denominator) = staking_config::get_reward_rate(&staking_config::get());
```

Comment:
It seems that `staking_config::get_reward_rate` does not exist on the head of `main`. Am I missing something? https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/staking_contract.move

Reply:
It's in the staking_config.move

Reply:
Ah, I misread `staking_config` as `staking_contract`. Thank you!